### PR TITLE
Simplified calo pusle shape utility

### PR DESCRIPTION
**Pull request by bechenard**
That shouldn't change anything, but third order effects might induce tiny shifts in the calo reco

### CODE REVIEW

**Comment by ryuwd** [comment]
@FNALbuild test

**Comment by ryuwd** [comment]
Apologies for the bot spam.

File: Mu2eUtilities/src/CaloPulseShape.cc [hunk 1]
```diff
@@ -43,19 +42,6 @@ namespace mu2e {
...
-       while(iend<pulseShape.GetNbinsX()) {if (pulseShape.GetBinContent(iend)  <fitlevel) break; ++iend;}
-
-       pulseShape.Fit("pol6","q0","",pulseShape.GetBinCenter(istart), pulseShape.GetBinCenter(iend));
-       TF1 *funPeak = pulseShape.GetFunction("pol6");
-       while (istart < iend) {if (fabs(pulseShape.GetBinContent(istart)-funPeak->Eval(pulseShape.GetBinCenter(istart))) < 0.001) break; ++istart;}
-       while (iend > istart) {if (fabs(pulseShape.GetBinContent(iend)-funPeak->Eval(pulseShape.GetBinCenter(iend))) < 0.001) break; --iend;}
-       for (int i=istart; i<iend;++i) pulseShape.SetBinContent(i,funPeak->Eval(pulseShape.GetBinCenter(i)));
-
        // Cache histogram content into vector and shift waveform (see note), 
        // calculate the number of bins for the digitized waveform
```
Comment (brownd1978) [inline Mu2eUtilities/src/CaloPulseShape.cc:46]: do these simplifications go together with the waveform change?

**Comment by bechenard** [comment]
Yes, the deleted lines were there to smooth the top of the waveform. I 
think it is cleaner to smooth the waveform externally (so this is the 
new waveform file) and remove this code.

**Review by brownd1978 — APPROVED** [review]

**Comment by kutschke** [comment]
@FNALbuild run build tests

**Comment by kutschke** [comment]
Adding a real comment on PR #440 seems to have moved that one along.  Let's see if adding a silly comment helps this one?

**Comment by kutschke** [comment]
Bertrand:  I am not sure why the CI on this is stuck and don't have time to look into it now.  When you get a chance please commit a do-nothing change to this branch (update a comment or something).  The re run the tests.

**Comment by kutschke** [comment]
Hmm.  Adding my last comment triggered a state update which I think will let me run the tests.  This time for sure ....

**Comment by kutschke** [comment]
@FNALbuild run build tests

**Comment by kutschke** [comment]
@FNALbuild run build tests

**Comment by brownd1978** [comment]
@FNALbuild run build tests

**Pull request merged**